src/SDKs/ContainerInstance/Management.ContainerInstance/Generated/ContainerInstanceManagementClient.cs

```diff
@@ -301,10 +342,10 @@ private void Initialize()
             ContainerGroups = new ContainerGroupsOperations(this);
             Operations = new Operations(this);
             ContainerGroupUsage = new ContainerGroupUsageOperations(this);
-            ContainerLogs = new ContainerLogsOperations(this);
```

Comment:
These look like breaking changes. Please bump nuget package version to `2.0.0` in the csproj and AssemblyInfo.cs files